pkg/monitor/clustermonitor.go

```diff
@@ -22,12 +22,12 @@ import (
 
 // Find returns a bool if the item exists in the given slice
 func Find(slice []types.ManagedClusterInfo, val types.ManagedClusterInfo) (int, bool) {
-    for i, item := range slice {
-        if item.Namespace == val.Namespace {
-            return i, true
-        }
-    }
-    return -1, false
+	for i, item := range slice {
+		if item.Namespace == val.Namespace {
+			return i, true
+		}
+	}
+	return -1, false
```

Comment:
I think the golang best practice is to return `nil`.  I'm basing my assumption on the implementation of other functions, would have to confirm.

**This one is obviously nit picking, it works as implemented.**